tests/webapp_t.py

```diff
@@ -228,6 +228,22 @@ def test_many_slash_info_request(self):
         self.assertEqual(loris_request.request_type, u'info')
         self.assertEqual(loris_request.ident, encoded_identifier)
 
+    def test_template_delimiter_request(self):
+        identifier = u'a:foo|bar'
+        encoded_identifier = u'a%3Afoo%7Cbar'
+        #image request
+        path = u'/%s/full/full/0/default.jpg' % identifier
+        req = _get_werkzeug_request(path)
+        loris_request = webapp.LorisRequest(req, False, None)
+        self.assertEqual(loris_request.request_type, u'image')
+        self.assertEqual(loris_request.ident, encoded_identifier)
+        #info request
+        path = u'/%s/info.json' % identifier
+        req = _get_werkzeug_request(path)
+        loris_request = webapp.LorisRequest(req, False, None)
```

Comment:
Minor: please could you use keyword arguments here (and on L237) so it’s clearer what these values mean? i.e.

```python
webapp.LorisRequest(req, redirect_id_slash_to_info=False, proxy_path=None)
```

(Should we consider defaulting those values?)

Reply:
added keyword arguments, and then made default values and removed the arguments from the test completely.